### ISSUE TITLE
[SUGGESTION] Media Scanner disabler

### ISSUE THREAD

**Issue by andiandi13** [vanilla-music/vanilla#947]
I think one of the most annoying thing, is to see all of our library disappear and that we have to rescan all the memory.

Currently I do many operations on my device that require that I remove my SD Card while it's powered on.

And of course only songs stored in internal memory remain in Vanilla.

So I suggest that you add a simple checkbox inside Media Library menu, that 'Disable Scanner', so that our library and playlists always stay the same even if we delete songs, remove the SDCard, move files etc... 

It'd be very useful and helpful to prevent rescanning the entire library.

**Comment by andiandi13** [comment]
@adrian-bl Is that something that you would do, and is that easy to implement ?

**Comment by adrian-bl** [comment]
Yes, that sounds like a handy feature which is simple to implement (just don't have much time lately to work on Vanilla)

**Comment by mase76** [comment]
That would solve the annoying behaviour, that playlists become empty, when the external sdcard ist not present directly after boot.

**Comment by andiandi13** [comment]
@adrian-bl Anything new about that possibility ?
